c11047543.lua

```diff
@@ -36,13 +36,13 @@ function c11047543.target(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
 	Duel.SetOperationInfo(0,CATEGORY_SPECIAL_SUMMON,nil,1,tp,LOCATION_EXTRA)
 end
 function c11047543.operation(e,tp,eg,ep,ev,re,r,rp)
-	local g=Duel.GetChainInfo(0,CHAININFO_TARGET_CARDS)
+	local g=Duel.GetTargetsRelateToChain()
+	if g:GetCount()~=2 or Duel.SendtoGrave(g,REASON_EFFECT+REASON_RETURN)==0 then return end
```

Comment:
shouldn't be SendtoGrave()~=2 ?

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1966/commits/8080c05ce0ebdfeeabe372ccd68f99796ccb2f2f fixed